templates/index.html

```diff
@@ -272,25 +272,7 @@ <h4 class="title"></h4>
             <p class="mainStreet" style="margin:0"></p>
             <p class="secondaryStreet" style="margin:0"></p>
             <p class="junction" style="margin:0"></p>
-            <p class="involved_type" style="margin:0"></p>
```

Comment:
I see. Still, perhaps it would make sense to write the HTML as a template,
so that in the code you just fill in the fields and append it. See e.g. the
sidebar entry template.

On Tue, Jun 30, 2015, 14:07 omerxx <notifications@github.com> wrote:

> In templates/index.html
> <https://github.com/hasadna/anyway/pull/339#discussion_r33561643>:
>
> > @@ -272,25 +272,7 @@ <h4 class="title"></h4>
> >              <p class="mainStreet" style="margin:0"></p>
> >              <p class="secondaryStreet" style="margin:0"></p>
> >              <p class="junction" style="margin:0"></p>
> > -            <p class="involved_type" style="margin:0"></p>
>
> @danielhers <https://github.com/danielhers> Since the deleted fields
> belong to 'involved' and 'vehicles' data, a marker may contain several
> blocks of each. Hence the jQuery .append and the removal of the above
> fields
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hasadna/anyway/pull/339/files#r33561643>.
>